docs/en/wallet-provider-endpoint.rst

```diff
@@ -665,10 +665,10 @@ The JOSE header of the Wallet Unit Attestation JWT contains the following parame
       - REQUIRED. It MUST be set to ``key-attestation+jwt``
       - `OPENID4VC-HAIP`_.
     * - **trust_chain**
-      - REQUIRED. Sequence of Entity Statements that composes the Trust Chain related to the Wallet Provider.
+      - OPTIONAL. Sequence of Entity Statements that composes the Trust Chain related to the Wallet Provider.
       - `OID-FED`_ Section 4.3 *Trust Chain Header Parameter*.
     * - **x5c**
-      - OPTIONAL. Contains the X.509 public key certificate or certificate chain (:rfc:`5280`) corresponding to the key used to digitally sign the JWT.
+      - REQUIRED. Contains the X.509 public key certificate or certificate chain (:rfc:`5280`) corresponding to the key used to digitally sign the JWT.
```

Comment:
Need to be OPTIONAL since we already have kid REQUIRED. 
See https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#appendix-F.1
> The key attestation may use x5c, kid or trust_chain (as defined in [Appendix F.1](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#jwt-proof-type)) to convey the public key and the associated trust mechanism to sign the key attestation.